src/Raven.Server/Commercial/SetupManager.cs

```diff
@@ -622,11 +652,13 @@ private static string GetServerUrlFromCertificate(X509Certificate2 cert, SetupIn
                             throw new InvalidOperationException($"Could not generate a client certificate for '{setupInfo.Domain.ToLower()}'.", e);
                         }
 
+                        RegisterClientCertInOs(onProgress, progress, clientCert);
```

Comment:
I'm not really happy with this, at the same time that I agree that this is a good idea.
Note that this will only work on the local machine, won't work with Docker, for example.